### DISCUSSION TITLE
QuickBMS: fails to install

### DISCUSSION THREAD

**Discussion by ypurpl** [ScoopInstaller/Scoop#6102] [Help and Questions]
Manifest: https://github.com/vvirtues/bucket/blob/main/bucket/quickbms.json
Once this is decent i'll make a request to add to Extras (not sure if it fits main criteria)
When I install, the binaries get put here:
![image](https://github.com/user-attachments/assets/d7524801-0a10-48c9-9288-f13ad5e19a1f)
and the terminal suddenly quits 
![image](https://github.com/user-attachments/assets/b19f3909-dca1-44e2-ae01-7c45f90ff7a3)
it doesnt seem to do the extract_to I specified, so the other files are placed in root (unintended):
![image](https://github.com/user-attachments/assets/73364e2b-c1fe-4e09-8966-74ace0e9e9ee)
could I get some help with this?

**Comment by ypurpl** [comment]
ok I fixed it, but now the scripts shortcut refuses to be made, even though the directory structure is right. Can scoop shortcuts link to a folder?

**Reply by HUMORCE** [reply]
Scoop does not support creating shortcuts to folder.

```
    "pre_install": [
        "\"explorer \"\"$dir\\scripts\"\"\" > \"$dir\\scripts.ps1\""
    ],
    "shortcuts": [
        [
            "scripts.ps1",
            "QuickBMS/QuickBMS Scripts",
            "",
            "quickbms.exe"
        ]
    ]
```

If you want a true shortcut, you should create it manually. The relevant functions of Scoop Core can be used as a reference. e.g. https://github.com/ScoopInstaller/Scoop/discussions/6046#discussioncomment-10165738
